src/OrchardCore.Modules/OrchardCore.Contents/TagHelpers/ContentItemTagHelper.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using OrchardCore.DisplayManagement;
+using OrchardCore.DisplayManagement.TagHelpers;
+
+namespace OrchardCore.Contents.TagHelpers
+{
+    [HtmlTargetElement("contentitem")]
+    [HtmlTargetElement("contentitem", Attributes = PropertyPrefix + "*")]
```

Comment:
@sebastienros I am not sure this line is required. It was working without it. (copied from ShapeTagHelper but not present in MenuTagHelper)

Reply:
This is only necessary for mapping strongly typed attributes on any name with a local dictionary, so I think you don't need that here.